x-pack/test/cloud_security_posture_functional/page_objects/findings_page.ts

```diff
@@ -52,50 +52,52 @@ export function FindingsPageProvider({ getService, getPageObjects }: FtrProvider
       testSubjects.click(type === 'failed' ? 'distribution_bar_failed' : 'distribution_bar_passed'),
   };
 
-  const table = {
-    getElement: () => testSubjects.find('findings_table'),
+  const latestFindingsTable = {
+    getElement: (selector: string) => testSubjects.find(selector),
 
-    getHeaders: async () => {
-      const element = await table.getElement();
+    getFindingsContainer: () => testSubjects.find('findings_container'),
+
+    getHeaders: async (selector: string) => {
+      const element = await latestFindingsTable.getElement(selector);
       return await element.findAllByCssSelector('thead tr :is(th,td)');
     },
 
     getColumnIndex: async (columnName: string) => {
-      const headers = await table.getHeaders();
+      const headers = await latestFindingsTable.getHeaders('findings_table');
```

Comment:
`findings_table` is assigned to all three tables, so it's both risky to query for it and misleading to read it. instead we should change the test id and from within each page object, make sure we're querying its own table